source/_components/sensor.google_travel_time.markdown

```diff
@@ -54,7 +54,7 @@ name:
   required: false
   type: string
 travel_mode:
-  description: "You can choose between: `driving`, `walking`, `bicycling` or `transit`."
+  description: "You can choose between: `driving`, `walking`, `bicycling` or `transit`. This method is now deprecated, use `mode` under `options`."
```

Comment:
Are you sure that it is deprecated? In [the code](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/google_travel_time.py#L55) is see that it is still an optional variable